### ISSUE TITLE
Move the HOT Summit logo to the side or make it smaller

### ISSUE THREAD

**Issue by cgiovando** [hotosm/hot-summit#3]
At the moment the summit logo and the white background around it take up half the screen. It should be moved either to the side, to have content next to it, or maybe up on the main menu bar.

**Comment by danbjoseph** [comment]
osmgeoweek has a splash img behind the main logo (globe and agriculture themed watermark type thing). instead of shrinking the logo, do we want to keep the relatively stripped down landing page but include some sort of backdrop so that it's not so much white space? one thought... same globe in the middle but in the corners a satellite (imagery) and something utilizing stuff similar to the un ocha urban/rural icon? colors changed to match the page palette. 

![urbanrural-01](https://cloud.githubusercontent.com/assets/4806884/6198482/7804adce-b43f-11e4-8df7-5735373dc963.png)
![screen shot 2015-02-14 at 11 45 51 am](https://cloud.githubusercontent.com/assets/4806884/6198492/dde940dc-b43f-11e4-9864-2377e126319f.png)

![2014-splash-food-2](https://cloud.githubusercontent.com/assets/4806884/6198488/bb3dec90-b43f-11e4-9260-93314b077a8e.png)

**Comment by mikelmaron** [comment]
I like the idea @danjoseph. It's at least the easiest. Could also take the padding off the header to save some screen real estate. \* Mikel Maron * +14152835207 @mikel s:mikelmaron 

```
 On Friday, February 13, 2015 7:53 PM, Dan Joseph <notifications@github.com> wrote:
```

 osmgeoweek has a splash img behind the main logo (globe and agriculture themed watermark type thing). instead of shrinking the logo, do we want to keep the relatively stripped down landing page but include some sort of backdrop so that it's not so much white space? one thought... same globe in the middle but in the corners a satellite (imagery) and something utilizing stuff similar to the un ocha urban/rural icon? colors changed to match the page palette.

**Comment by danbjoseph** [comment]
This was my quick work-in-progress, but it's pretty amateurish looking and I think the stuff on the sides gets cut off pretty quick at smaller screen sizes.
![hot-summit_backdrop](https://cloud.githubusercontent.com/assets/4806884/6202253/73ee4d46-b519-11e4-9f2a-489b2a063cbc.png)

**Comment by cgiovando** [comment]
I like your proposal, can you please add in the background? White is OK for mobile phone rendering, but in a desktop browser it's just taking 1/3 of the screen space for nothing. Maybe we can still push the logo to the side and add intro text - a description of what the event is about - right next to it?

**Comment by bgirardot** [comment]
I don't think that most recent example looks amateurish at all, I think it looks really cool.
